packages/react-theme-provider/src/Compat/ThemeProvider.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
-import { ThemeProvider as ReactThemeProvider, ThemeProviderProps } from '@fluentui/react-theme-provider';
-import { useCustomizationSettings } from '../Utilities';
+import { ThemeProvider as ReactThemeProvider, ThemeProviderProps } from '../index';
+import { useCustomizationSettings } from '@uifabric/utilities';
 import { convertLegacyTheme } from './convertLegacyTheme';
-import { ITheme } from '../Styling';
+import { ITheme } from '@uifabric/styling';
 
-export { ThemeProviderProps, useTheme } from '@fluentui/react-theme-provider';
+export { ThemeProviderProps, useTheme } from '../index';
```

Comment:
nit: You can now put these and the imports on line 2 in the same line.

Reply:
How? This is an export statement. Also I don't want to get too far down the rabbit hole of cleaning up the code while moving it.

Reply:
Oh, my bad, thought it was an input as well. Resolving.

Reply:
Np :)